### DISCUSSION TITLE
posts/how-to-setup-this-blog

### DISCUSSION THREAD

**Discussion by giscus** [mildronize/blog-v8#33] [Blog Comment]
# posts/how-to-setup-this-blog

This blog created by Jekyll that is ruby scripts to transforms plain text into static html website and blogs and hosted on Github Page that free hosting support Jekyll.

https://www.thadaw.com/posts/how-to-setup-this-blog?id=we3xr8r

**Comment by mildronize** [comment]
Test comment
